### DISCUSSION TITLE
Installing Surface kernel using WindowsFX

### DISCUSSION THREAD

**Discussion by coufeyrac** [linux-surface/linux-surface#675] [Q&A]
Has anyone tried WindowsFX on the Surface Pro 5/2017? I'm trying it in hopes the Windows aspects on top of Ubuntu 20 will allow the cameras and pen to work properly.

I can't seem to install the kernel, though - I keep getting errors stating that the IP address is blocked or "ignoring file 'linux-surface.listsudo' in directory '/etc/apt/sources.list.d/' as it has an invalid filename extension"

Thank to everyone for any help!

**Comment by SamLukeYes** [comment]
Maybe you've hit https://github.com/linux-surface/linux-surface/issues/625
`/etc/apt/sources.list.d/linux-surface.listsudo` looks like a file created by mistake. If the linux-surface repo is recognized by apt already (should appear in the output of `apt update`), it can be safely removed.

**Reply by coufeyrac** [reply]
Thanks. I already tried to remove it but no luck. It comes back saying it doesn't exist... odd, no?
